qlib/contrib/model/pytorch_nn.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         self,
         input_dim,
         output_dim,
-        layers=(256, 256, 128),
+        layers=(256, 512, 768, 1024, 768, 512, 256, 128, 64),
```

Comment:
Will the change of NN structure affects the performance greatly?

Reply:
Yes